splunk-devops/src/main/java/com/splunk/splunkjenkins/CustomLoggerItem.java

```diff
@@ -0,0 +1,80 @@
+package com.splunk.splunkjenkins;
+
+import hudson.Extension;
+import hudson.logging.LogRecorder;
+import hudson.logging.LogRecorderManager;
+import hudson.model.Describable;
+import hudson.model.Descriptor;
+import hudson.util.FormValidation;
+import hudson.util.ListBoxModel;
+import jenkins.model.Jenkins;
+import org.kohsuke.stapler.DataBoundConstructor;
+import org.kohsuke.stapler.QueryParameter;
+
+public class CustomLoggerItem implements Describable<CustomLoggerItem> {
+    private static final java.util.logging.Logger LOG = java.util.logging.Logger.getLogger(CustomLoggerItem.class.getName());
+
+    private static transient final LogRecorderManager logRecorderManager = Jenkins.getInstance().getLog();
```

Comment:
this depend on LogRecorderManager, which adds a https://github.com/jenkinsci/jenkins/blob/75b4b31efd3f3340805497b6b23ff4b0cb32d2f3/core/src/main/java/hudson/util/RingBufferLogHandler.java automatically, that means if you want to sent to splunk verbose level log, it has to be logged to RingBufferLogHandler too, unnecessary performance burden.